demisto_sdk/commands/common/update_id_set.py

```diff
@@ -2293,15 +2341,15 @@ def sort(data):
     return data
 
 
-def update_excluded_items_dict(excluded_items_from_id_set: dict, excluded_items_to_add: dict,
-                               excluded_items_by_type: dict):
+def update_excluded_items_dict(excluded_items_by_pack: dict, excluded_items_by_type: dict,
+                               excluded_items_from_iteration: dict):
```

Comment:
`excluded_items_from_iteration` - the original name was better.
the argument name should not be affected by the calling scope